src/views/Admin/Admin.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { useState } from 'react';
+import DogForm from '../../components/DogForm';
+import { createDog } from '../../services/dog';
+
+export default function Admin() {
+  const [dog, setDog] = useState({});
+
+  const updateDogState = (key, value) => {
+    dog[key] = value;
+    setDog({ ...dog });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    await createDog(dog);
+    if (dog !== '') {
```

Comment:
The default value for your dog state is an object. Empty objects are not falsely so this won't work as intended.